<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transformaciones</class>
 <widget class="QDialog" name="Transformaciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transformaciones</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>200</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60, 60, 60);</string>
   </property>
   <widget class="QLabel" name="titlLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Transformacion</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tx_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Tx:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinTx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinTy">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:  12pt &quot;Arial&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="ty_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Ty:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>üîº</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>üîΩ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>‚óÄÔ∏è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>‚ñ∂Ô∏è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Reiniciar </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinSy">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:  12pt &quot;Arial&quot;;</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sx_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Sx:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinSx">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sy_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Sy:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="escalarButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Escalar</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="thetaSpin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>45</number>
    </property>
   </widget>
   <widget class="QLabel" name="thetha">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Theta:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="rotationButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Rotar</string>
    </property>
   </widget>
   <widget class="QLabel" name="tx_label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Px:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPy">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:  12pt &quot;Arial&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>-250</number>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
   </widget>
   <widget class="QLabel" name="ty_label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Py:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>-250</number>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
   </widget>
   <widget class="QPushButton" name="animacion1Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="animacion2Button">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>430</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
